### PR TITLE
Issue #190: Fix split pane overflow scroll and headers

### LINKED ISSUE #190: Fix bugs relating to GUI display and window
To address all bugs with regards to default size when the GUI app is started, as well as how the display works with the minimisation of screens.

### PR DESCRIPTION
**Pull request by charmainehly**
Currently, minimising the screen will cause some text misalignment and truncation of information.

This might affect the user's ease of viewing.

Let's include the following changes:
* a scroll pane in the middle panel to allow overflow scrolling
* headers for each panel to indicate each panel's use
* larger default window size for the gui app
* extend the feedback box to have a larger minimum height

<br> **Previous UI upon minimisation:**
Screenshot 2022-03-28 at 2 27 59 AM

<br> **Updated UI upon minimisation:**
Screenshot 2022-03-28 at 12 16 57 PM

_Note to reviewers: appreciate your help to double check the default viewing dimensions! :) Feel free to leave comments on removing/modifying any of the above changes, or modifying them at your end after the merge!_

Closes #190

### CODE REVIEW

File: src/main/java/seedu/address/MainApp.java [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
     @Override
     public void init() throws Exception {
-        logger.info("=============================[ Initializing AddressBook ]===========================");
+        logger.info("=============================[ Initializing TalentAssistant ]===========================");
```
Comment (domlimm) [inline src/main/java/seedu/address/MainApp.java:56]: Good catch!

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -89,7 +89,7 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             addressBookOptional = storage.readAddressBook();
             interviewListOptional = storage.readInterviewSchedule();
             if (!addressBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample AddressBook");
+                logger.info("Data file not found. Will be starting with a sample TalentAssistant");
```
Comment (domlimm) [inline src/main/java/seedu/address/MainApp.java:92]: Again, nice catch! 👍🏻

**Review by domlimm — APPROVED** [review]
Spot on! Thanks for fixing the bugs!

We gotta go bonkers on the resizing of the window on Wednesday.

Perhaps, we could consider disabling the window resize? Hopefully I can remember raising in up in meeting @charmainehly.

Nice! Approved! 😄

**Review by tiewweijian — APPROVED** [review]
LGTM ! I think the UI looks cleaner and the window is definitely bigger ! 
I think in the next iteration we could possible change `Candidates List's` interface too !

**Pull request merged**